skyportal/handlers/api/sources_confirmed_in_gcn.py

```diff
@@ -184,20 +205,20 @@ def post(self, dateobs):
                   source_id:
                     type: string
                     description: The source_id of the source to confirm or reject
-                  confirmed_or_rejected:
+                  confirmed:
                     type: boolean
-                    description: Whether the source is confirmed or rejected
+                    description: Whether the source is confirmed (True) or rejected (False)
                   start_date:
                     type: string
-                    description: The start date (min first detected in photstat) of the source, as an arrow parseable string
+                    description: First detection of source(s) after start_date, as an arrow parseable string
```

Comment:
I would go with something like "return sources with a first detection after start_date..."

Reply:
This is in the post method which does not return anything. It's just an argument used to query sources in a localization (we use this time constraint as sources detected way before or way after an event have nothing to do with it).

Reply:
Right, return might be the wrong word but maybe "choose sources with..."